sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py

```diff
@@ -84,20 +85,20 @@ def close(self):
         self._generated_client.close()
 
     def register_schema(
-        self, group_name, name, content, serialization_type, **kwargs
+        self, group_name, name, schema_definition, format, **kwargs  # pylint:disable=redefined-builtin
     ):
-        # type: (str, str, str, Union[str, SerializationType], Any) -> SchemaProperties
+        # type: (str, str, str, Union[str, SchemaFormat], Any) -> SchemaProperties
         """
         Register new schema. If schema of specified name does not exist in specified group,
         schema is created at version 1. If schema of specified name exists already in specified group,
         schema is created at latest version + 1.
 
         :param str group_name: Schema group under which schema should be registered.
         :param str name: Name of schema being registered.
-        :param str content: String representation of the schema being registered.
-        :param serialization_type: Serialization type for the schema being registered.
-         For now Avro is the only supported serialization type by the service.
-        :type serialization_type: Union[str, SerializationType]
+        :param str schema_definition: String representation of the schema being registered.
+        :param format: Format for the schema being registered.
+         For now Avro is the only supported schema format by the service.
+        :type format: Union[str, SchemaFormat]
         :keyword content_type: The content type of the request. Default value is 'application/json'.
         :paramtype content_type: str
```

Comment:
let's pull out the docstring for `content_type` first.
I remember the service team mention that they want to reserve it for schema content type/format.